packages/perseus/src/widgets/expression/expression.tsx

```diff
@@ -322,56 +325,30 @@ function getStartUserInput(): PerseusExpressionUserInput {
     return "";
 }
 
+function getOneCorrectAnswerFromRubric(
```

Comment:
Just a nit, moved it because I think it's nicer when they're not declared in the export object.